librepo/gpg.c

```diff
@@ -393,7 +393,7 @@ lr_gpg_list_keys(gboolean export_keys, const char *home_dir, GError **err)
             subkey = subkey->next;
         }
          // All subkeys in the list except the last one are followed by another subkey
-        if (subkeys->len > 0) {
+        if (subkeys->len > 1) {
```

Comment:
May I ask you why the first subkey is skipped? It was skipped even by original code. The description says something about the last key, but it skips the first one. May I ask you to enhance the comment?

Reply:
The first key is not skipped. But if there is only one key in the list, the first key is also the last key. It is unnecessary to continue with the following cycle. It would run zero times anyway. So changing `keys->len > 0` to `keys->len > 1` is just an optimization.

Reply:
May I ask you to enhance comment in lines 395 and 422. I would like to see that the code bellow mark keys that are followed by ` // Mark all subkeys in the list except the last one that they are followed by another subkey`

Reply:
OK. I modified the comment.